### DISCUSSION TITLE
The game crashed whilst initializing game Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors,

### DISCUSSION THREAD

**Discussion by SkyressOW** [FabricMC/fabric-api#1399] [FAQ]
So my friend wanted me to join their fabric server and sent me everything I needed to go onto it, but after a couple of times on the server, I couldn't get back on and The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'alloy_forgery'!
Exit Code: -1 happened.

I've never used Fabric before but I followed what I had seen for how to download everything, soooo idk what I did wrong.

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'alloy_forgery'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	Suppressed: java.util.NoSuchElementException
		at java.util.Scanner.throwFor(Scanner.java:862)
		at java.util.Scanner.next(Scanner.java:1371)
		at wraith.alloy_forgery.utils.Config.readFile(Config.java:60)
		at wraith.alloy_forgery.utils.Config.init(Config.java:29)
		at nourl.mythicmetals.config.MythicForgeConfig.createAlloys(MythicForgeConfig.java:14)
		at nourl.mythicmetals.MythicMetals.onInitialize(MythicMetals.java:50)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient$$Lambda$2341/869335753.accept(Unknown Source)
		at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
		... 11 more
Caused by: java.util.NoSuchElementException
	at java.util.Scanner.throwFor(Scanner.java:862)
	at java.util.Scanner.next(Scanner.java:1371)
	at wraith.alloy_forgery.utils.Config.readFile(Config.java:60)
	at wraith.alloy_forgery.utils.Config.init(Config.java:29)
	at wraith.alloy_forgery.AlloyForgery.onInitialize(AlloyForgery.java:20)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient$$Lambda$2341/869335753.accept(Unknown Source)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
	... 11 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1292524456 bytes (1232 MB) / 2147483648 bytes (2048 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 12
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		adorn: Adorn 1.13.0+1.16.5
		alloy_forgery: Alloy Forgery 1.0.7
		antiqueatlas: Antique Atlas 5.4.4-fabric
		appleskin: AppleSkin 1.0.11
		artifice: Artifice 0.14.6+1.16.5
		autoconfig1u: Auto Config v1 Updated 3.3.1
		bedspreads: Bedspreads 1.3.1-1.16.4
		bitsandchisels: BitsAndChisels 2.1.2
		blue_endless_jankson: jankson 1.2.0
		cardinal-components-base: Cardinal Components API (base) 2.7.9
		cardinal-components-entity: Cardinal Components API (entities) 2.7.9
		cardinal-components-item: Cardinal Components API (items) 2.7.9
		chromaconcrete: Chroma Concrete 1.1.0
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.5.47
		cloth-config2: Cloth Config v4 4.11.15
		compact-storage: CompactStorage 4.1.0
		cosmetic-armor: Cosmetic Armor 1.0.0
		curios: Curios API 0.0.13-1.16.5
		customportalapi: customportalapi 0.0.1-beta18-1.16
		enchant_giver: Enchant Giver 1.0.3
		extraorigins: Extra Origins 1.16.5-9
		fabric: Fabric API 0.30.3+1.16
		fabric-api-base: Fabric API Base 0.2.1+9354966b7d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.0.10+ca58154a7d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.6+9354966b7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-furnaces: Fabric Furnaces 1.3.3-1.16.4
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.3+ca58154a7d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-language-kotlin: Fabric Language Kotlin 1.5.0+kotlin.1.4.31
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.4+9354966b7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabricloader: Fabric Loader 0.11.3
		fallflyinglib: FallFlyingLib 1.1.0
		fatxporbs: Fat Experience Orbs 0.0.9
		flesh2leather: Flesh2Leather 1.0.2
		gobber2: Gobber2 2.3.71
		golemsgalore: Golems Galore 0.2.0+mc.1.16.2
		inventorysorter: Inventory Sorter 1.7.7-1.16
		jankson: Jankson 3.0.1+j1.2.0
		java: Java HotSpot(TM) 64-Bit Server VM 8
		kyrptconfig: Kytpt Config 1.1.6-1.16
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 3.3.2+1.16.4
		magna: Magna 1.5.1-1.16.5
		maybe-data: Maybe data 1.0.0-1.16.2
		mealapi: Meal API 0.2+1.16.4
		minecraft: Minecraft 1.16.5
		mythicmetals: Mythic Metals 0.9.9
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.4.31
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.4.31
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.4.31
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.4.31
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.4.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.4.2
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.1.0
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.1.0
		origins: Origins 0.6.1
		origins-classes: Origins: Classes 1.1.1
		pehkui: Pehkui 1.9.0+21w10a
		petowner: Pet Owner 1.7.1-1.16.2+
		playerabilitylib: Pal 1.2.1
		pugh_lib: Pugh Lib 1.1.0
		reach-entity-attributes: Reach Entity Attribute 1.0.1
		reroll: Reroll 1.2.0-1.16.5
		roughlyenoughitems: Roughly Enough Items 5.10.184
		roughlyenoughitems-api: REI (API) 5.10.184
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.10.184
		roughlyenoughitems-runtime: REI (Runtime) 5.10.184
		secretrooms: Secret Rooms 0.9.3+1.16.5
		slimeorigin: Slimeorigin 2.0.0
		stacc: Stacc 1.0.1
		staffofbuilding: Staff of Building 1.2.5-1.16.5
		static-content: Static Content 1.0.1-1.16.2
		staticdata: Static Data 1.1.2
		strawdummy: Straw Dummy 1.4.0
		structurized: Structurized 1.1.0
		transparent_cosmetics: Transparent Cosmetics 1.1.1
		vanilla-hammers: Vanilla Hammers 3.0.2-1.16.5
		vanillaexcavators: Vanilla Excavators 1.1.0
		villagerhats: Villager Hats fabric-1.16.4-1.0.1
		waystones: Waystones 1.1.1.1
		xlpackets: XLPackets 1.1
		xp_storage: XP Storage 1.4+1.16.5
	Launched Version: fabric-loader-0.11.3-1.16.5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: NO CONTEXT
	GL Caps: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>

**Comment by liach** [comment]
Try delete your config for alloy forgery mod. It looks like it crashes because it doesn't handle broken config format.

**Comment by Linguardium** [comment]
CrowdinTranslate is trying to patch into the same place as fabric api and failing. they should stop doing that. I suggest moving to the last supported fabric api that they support until they fix their mod and use the fapi stuff instead
